dataland-inbound-proxy/templates/dataland-development.template

```diff
@@ -46,4 +46,6 @@ server {
 
     # additional config
     include utils/general.conf;
+    include utils/error.conf;
+
```

Comment:
Der include fehlt in dem production template

Reply:
In production template eingebaut